src/value/element_stream_reader.rs

```diff
@@ -481,9 +481,9 @@ mod reader_tests {
         next_type(reader, IonType::Struct, false);
         reader.step_in()?;
         next_type(reader, IonType::Integer, false);
-        assert_eq!(reader.field_name()?, text_token("foo"));
+        assert_eq!(reader.field_name()?, Symbol::owned("foo"));
```

Comment:
🗺️ The `text_token` method was a vestige of the old `IonSymbolToken` trait and API.